bat/tests/reset-command/run.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+
+# shared test functions
+load ../../lib/mixerlib
+
+setup() {
+  global_setup
+}
+
+@test "reset" {
+  mixer-init-stripped-down 27310 10
+  mixer build all --format 30 --native
+  mixer versions update --upstream-version 30000
+  mixer build upstream-format --new-format 28 --native
+  sudo mixer versions update --upstream-version 30800
+  mixer build upstream-format --new-format 20 --native
```

Comment:
Any reason why you are doing format bumps in this test?

Reply:
To validate if upstreamversion file content is updated. Upstreamversion file will only only change during format bump

Reply:
That partially correct. `upstreamversion` will update on a build, and a since format bump will perform two builds, it ends up updating the value. Since you are not going through an upstream format barrier here, just a `mixer build all` will suffice.
With that being said, I believe it would be much better if `build upstream-format` was smart enough to run a single build if a format bump is not needed. Instead of defaulting to "always 1 format barrier will be crossed", the command should first check if a bump is really needed and, if not, if should behave as `mixer build all`. This will make the process of updating upstream version more streamlined. This is not for this PR though, it should be its own patch.

Reply:
got it. Updated the PR.